ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3110,3 +3110,15 @@ literals:
 - ultraviolet_irradiance_sensor:
     fixed_min: 0.0
     flexible_max: 2000.0
+
+- supply_air_linearvelocity_sensor:
+    fixed_min: 0.0
+    flexible_max:  25.4000508
+
+- supply_fan_rotationalvelocity_sensor:
+    fixed_min: 0.0
+    flexible_max:  25.4000508
+
+- exhaust_fan_rotationalvelocity_sensor:
```

Comment:
It is unusual to see `rotationalvelocity` sensors. Are there more common measures like `speed_percentage` or `speed_frequency` that are available in your system?

Reply:
The rotational velocity was used because the units on the reporting field are rpm, is this okay?